controllers/host/platformnetworks.go

```diff
@@ -337,6 +370,22 @@ func (r *HostReconciler) ReconcileAddrPoolResource(client *gophercloud.ServiceCl
 
 	validation_result := r.ValidateAddressPool(network_instance, addrpool_instance, system_info)
 
+	// Secondary stacks for cluster-* should always be configured
+	// after making sure OAM dual-stack is configured to avoid
+	// networking issues.
+	if network_instance.Spec.Type == cloudManager.ClusterHostNetworkType ||
+		network_instance.Spec.Type == cloudManager.ClusterPodNetworkType ||
+		network_instance.Spec.Type == cloudManager.ClusterServiceNetworkType {
+
+		is_oam_dual_stack := r.IsOAMDualStack(client)
+
+		if len(network_instance.Spec.AssociatedAddressPools) == 2 &&
+			!is_oam_dual_stack {
+			msg := "Waiting for OAM dual-stack to be configured first."
```

Comment:
Need to clarify the error message: How to wait? 
1. What if the OAM address is not configured as dual-stack? Should we give the user a clear message about that 
2. If it is specified in the configuration, need to tell the user to wait for another round of reconciliation until the OAM is configured

Reply:
Done.